back/src/bsffs/typeDefs/bsff.objects.graphql

```diff
@@ -296,9 +296,13 @@ type BsffFicheIntervention {
   "Code postal du lieu où l'intervention a eu lieu."
   postalCode: String!
 }
+
+"Informations sur le détenteur d'un équipement"
 type BsffDetenteur {
-  "Entreprise détentrice de l'équipement."
-  company: FormCompany!
+  "Informations sur l'entreprise détentrice dans le cas où le détenteur est une entreprise"
+  company: FormCompany
+  "Informations sur la personne dans le cas où le détenteur est un particulier"
+  privateIndividual: PrivateIndividual
```

Comment:
Petite diff avec le bsda, qui met tout dans company avec un boolean isPrivateIndividual.
Idéalement on aurait sans dout du avoir qque chose comme `entity: FormCompany | PrivateIndividual`... Ca sera pour la v3 😆

Reply:
Ouais j'ai vu... mais je trouve un peu bancale le fait de mettre les infos du particulier sous `company`. 
À voir si on privilégie l'homogénéité ou une amélioration ici. @providenz @elishowk vous en pensez quoi ? Je crois qu'Elias tu avais un peu planché sur la généralisation de `company` aux particuliers, OMI, associations, etc.

Reply:
J'ai basculé sur le fonctionnement BSDA / BSDD